### ISSUE TITLE
Major world corruption: Ticking World crash

### ISSUE THREAD

**Issue by SlimefrogSocial** [ferreusveritas/Dynamic-Trees#466]
When entering new terrain, the entire server has been crashing for Exception Ticking World. This terrain was pregenerated with Chunk pregenerator and removing Dynamic Trees fixes the issue. These trees also look strange. They are floating and not connected at the root. Some trees are floating completely even though the trump is touching the ground.
Latest Dynamic Trees
BOP Dynamic Trees 1.4.4

Tried removing all dynamic trees addons, still crashes with only core mod in.

[crash-2020-11-16_22.07.55-server.txt](https://github.com/ferreusveritas/DynamicTrees/files/5551040/crash-2020-11-16_22.07.55-server.txt)

https://pastebin.com/tkcDsTFs

[latest.log](https://github.com/ferreusveritas/DynamicTrees/files/5551052/latest.log)

**Comment by SlimefrogSocial** [comment]
Continuing from:
https://github.com/Harleyoc1/DynamicTreesTraverse/issues/1
Next test:
Remove EVERY mod except DT core and see what happens

UPDATE:
Removing every mod except DT removes all dt trees as well, creative tab is there but all generated trees are gone... retrying with Traverse, BOP, and both compats installed 
UPDATE:
Still removed all trees, but my backup has me in creative so I am assuming I removed DT before backing it up... I will use the other back up and fly across creation to get to where the crash is (I was using an old one we just ran away from the corrupted area so I was using the one right outside it)
UPDATE:
All DT trees get removed when removing every mod but DT and same with removing all but it and its compats... Loading server with most mods removing only a few. Just making sure DT Trees are still there when loaded semi usually. If this doesnt work then I will load it fully normal and make sure I actually have a working backup...
UPDATE:
World Crashed so I assume the trees were still there, now to slowly remove mods till I find the problem...
Might be unrelated, but having zombie villager profession spam attached to the crash, here's the logs:
[debug-1.log.gz](https://github.com/ferreusveritas/DynamicTrees/files/5569493/debug-1.log.gz)

[2020-11-19-7.log.gz](https://github.com/ferreusveritas/DynamicTrees/files/5569494/2020-11-19-7.log.gz)

[crash-2020-11-19_15.16.15-server.txt](https://github.com/ferreusveritas/DynamicTrees/files/5569478/crash-2020-11-19_15.16.15-server.txt)
UPDATE!!!:
When removing the last grouping of mods, IT WORKED! I have a world (with almost nothing in it) that has DT and is no longer corrupted! Now to narrow down the ones that I removed to which one is breaking it...
